tests/integration/utils2.go

```diff
@@ -66,6 +66,10 @@ var (
 
 const subscriptionTimeout = 1 * time.Second
 
+// Instanstiating lenses is very expensive, and our tests do not benefit from a large number of them,
+// so we explicitly set it to a low value.
+const lensPoolSize = 2
```

Comment:
question: Can you tell me why instantiating is so expensive? When I look at the code path it's not so obvious.

todo: `Instantiating`

Reply:
I though I documented that somewhere in greater detail.  Lenses are wasm instances, so you are basically initing a full runtime environment.  It takes about 40ms each time on my laptop.

Reply:
This is documented on `lensLocker`, the type that uses this value and is responsible pooling lenses.

Reply:
Don't forget the typo :)

Reply:
:grin: My bad

- [ ] typo